### PR TITLE
Improve `readable-title-change-events` order

### PR DESCRIPTION
**Pull request by fregante**
- As suggested in https://github.com/refined-github/refined-github/pull/5703#issuecomment-1176568500

## Test URLs

This page

## Screenshot

Screen Shot 10

### CODE REVIEW

File: source/features/readable-title-change-events.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+/* TODO: Drop this whole rule when :has() is crossbrowser */
+.TimelineItem[id^='event-'] .TimelineItem-body :is(ins, del) {
+	display: block !important;
+}
+
+/*
+Test URLs
+https://github.com/refined-github/refined-github/pull/5793#event-6952233065
```
Comment (cheap-glitch) [inline source/features/readable-title-change-events.css:21]:
```suggestion
https://github.com/refined-github/refined-github/pull/5793#event-6952233065
https://github.com/refined-github/refined-github/pull/4030
```

**Review by cheap-glitch — APPROVED** [review]
This changes nothing in Firefox, but doesn't seem to break anything either.

File: source/features/readable-title-change-events.css (hunk 1 above)
Comment (fregante) [reply source/features/readable-title-change-events.css:21]: Correct. This is Safari-only at the moment as the label suggests.

Comment (fregante) [reply source/features/readable-title-change-events.css:21]: Good suggestion. That PR is enough

```suggestion
https://github.com/refined-github/refined-github/pull/4030#event-4385348325
```

**Pull request merged**